src/libs/ReportUtils.js

```diff
@@ -4043,6 +4044,21 @@ function getTaskAssigneeChatOnyxData(accountID, assigneeAccountID, taskReportID,
     };
 }
 
+/**
+ * Return the mention message of a list accounntID
+ * @param {Array<String>} accountIDs
+ * @returns {String}
+ */
+function getMentionMessage(accountIDs) {
```

Comment:
It's not really a message though... I think that this logic is an implementation detail of `getRoomChannelLogMemberMessage`. I'm not against extracting functions at all, it just seems more appropriate for this to be a local function _ inside_ `getRoomChannelLogMemberMessage`.

Having in my [the other thread](https://github.com/Expensify/App/pull/30898/files#r1383204202), I think we'll just have to slightly restructure the logic of building our string.